src/write.rs

```diff
@@ -143,6 +151,20 @@ impl FileOptions {
         self
     }
 
+    /// Set the compression level for the new file
+    ///
+    /// Range of values depends on compression method:
+    /// * `Deflated`: 0-9. `None` corresponds to 6
+    /// * `Bzip2`: 0-9. `None` corresponds to 6
+    /// * `Zstd`: 1-21. `None` corresponds to 3
+    /// * others: only `None` is allowed
+    /// `None` value specifies default compression level.
```

Comment:
I suggest to leave the information about compression level only in this place and remove duplication in other places. It will reduce complication in future maintenance.

Reply:
Agreed; removed other doc comments